### PR TITLE
Updated graphics.rst (Documentation)

### PR DESCRIPTION
**Pull request by Meet-Patel2580**
Updated the documentation for VPR graphics by adding new images of the UI as well as fixing grammatical errors and descriptions of the locations of various buttons.

#### Motivation and Context

The existing documentation displayed an older version of the user interface and was required to be updated with newer images as well as an updated description of how to use certain features.

#### Types of changes

- [ ] Bug fix (change which fixes an issue)
- [ ] New feature (change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

#### Checklist:

- [ ] My change requires a change to the documentation
- [x] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [x] All new and existing tests passed

### CODE REVIEW

File: doc/src/vpr/graphics.rst [hunk 1]
```diff
@@ -56,16 +54,28 @@ Visualizing Placement
...
     :align: center
 
     Placement with macros (carry chains) highlighted
 
-If the **Placement Macros** drop down is set, any placement macros (e.g. carry chains, which require specific relative placements between some blocks) will be highlighted.
-
 Visualizing Netlist Connectivity
 --------------------------------
-The **Toggle Nets** drop-down list toggles the nets in the circuit visible/invisible.
+The **Toggle Nets** drop-down list under the **Net Settings** tab toggles the nets in the circuit to be visible/invisible. Options include **Nets** and **Logical Connections**.
```
Comment (vaughnbetz) [inline doc/src/vpr/graphics.rst:72]: Look in the draw code to figure out what logical connections are and describe them here.  Send snippets of code to us by email if needed; can use git blame to see who added the logical connection lines in the menu.

**Review by vaughnbetz — CHANGES_REQUESTED** [review]

**Comment by vaughnbetz** [comment]
Looks good; one suggested change. Then ping me to merge.  Right after merging, check the web page to make sure all the images show up.

File: doc/src/vpr/graphics.rst (hunk 1 above)
Comment (vaughnbetz) [reply doc/src/vpr/graphics.rst:72]: Logical nets should be renamed --> suggest Primitive nets
Nets should be renamed --> suggest Cluster nets
Help should explain that showing the primitive nets showing connections within the cluster, and the primitive within the cluster that drives/receives them.  Cluster nets shows only the nets that go between clusters.

**Comment by vaughnbetz** [comment]
Another item to investigate (but should be another issue): can we show only the highlighted nets (fanin and fanout of selected block)?

**Comment by vaughnbetz** [comment]
Should also change the code to have "Set Net Transparency" --> Net Transparency (0 - 255)

**Comment by Meet-Patel2580** [comment]
All suggested changes made. @vaughnbetz , You may now merge this PR, and I will check right after to make sure the images on the documentation are showing up properly.

**Pull request merged**